Javascript/homework/responses/3.5-clock-hands.md

```diff
@@ -4,4 +4,42 @@
 
 #### Task 5: Designing the hands of your clock
 
-Style the hands so that the seconds hand is the longest and thinnest, and the hour hand is shortest and widest. These hands should also be set to rotate given a degree.
\ No newline at end of file
+Inside of your clock div define three divs - these will represent the hands on our clock.  For each div, set the class to "hand [type of hand]" and set the data-* attribute to `data-[type of hand]-hand`.  For example, the div tag for the hour hand would look like:
```

Comment:
"Inside of your clock div define three divs" I didn't quite understand this line.